eth_crowdfund_api/src/views/CampaignView.py

```diff
@@ -1,61 +1,62 @@
 from flask import request, json, Response, Blueprint, jsonify
 from ..models.Campaign import Campaign, CampaignSchema
-
+from marshmallow import ValidationError
+from sqlalchemy import exc
+from . import custom_response
 
 
 campaign_api = Blueprint('campaigns_api', __name__)
 campaign_schema = CampaignSchema()
 
 @campaign_api.route('/', methods=['POST'])
 def create():
-  req_data = request.get_json()
-  data = campaign_schema.load(req_data)
-  # data, error = campaign_schema.load(req_data)
-  # BUT WITH ERROR: the above code threw an error, currently ignoring errors
-  error = None
-  if error: 
-    return custom_response(error, 400)
+  try: 
+    req_data = request.get_json()
+    data = campaign_schema.load(req_data)
+  except ValidationError as err: 
```

Comment:
Catches errors raised by .load (deserialization) to have dynamic error messages for bad input data